ConfigSpace/hyperparameters.pyx

```diff
@@ -1123,20 +1159,17 @@ cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
         value = self._inverse_transform(value)
         return value
 
-    def _transform(self, vector: Union[np.ndarray, float, int]) -> Optional[Union[np.ndarray, int]]:
+    cpdef np.ndarray _transform_vector(self, np.ndarray vector):
+        if np.any(~np.isfinite(vector)):
```

Comment:
Is this check necessary? It's also conducted in the `_transform_vector` method of `nfhp`, right?

Reply:
Good point.